reportek/core/serializers.py

```diff
@@ -430,25 +438,91 @@ class Meta(FeedbackCommentSerializer.Meta):
         extra_kwargs = {'url': {'view_name': 'api:envelope-feedback-comment-detail'}}
 
 
+class EnvelopeObligationSpecSerializer(serializers.ModelSerializer):
+
+    obligation = serializers.SerializerMethodField()
+
+    @staticmethod
+    def get_obligation(obj):
+        return DynamicObligationSerializer(
+            obj.obligation_spec.obligation,
+            read_only=True,
+            fields=('id', 'title', 'terminated')
+        ).data
+
+    class Meta:
+        model = EnvelopeObligationSpec
+        fields = ('obligation_spec', 'obligation', 'reporting_cycle')
+
+
+class NestedEnvelopeObligationSpecSerializer(
```

Comment:
Is this still needed?

Reply:
No, it will be going away with the next push. Thanks for the catch.